### PR TITLE
Simplify finding current appeal

### PR DESCRIPTION
**Pull request by cvalarida**

### CODE REVIEW

File: src/js/claims-status/utils/appeals-v2-helpers.jsx [hunk 1]
```diff
@@ -250,8 +252,8 @@ export function getStatusContents(statusType, details = {}, name = {}) {
       const formattedSocDate = moment(details.lastSocDate, 'YYYY-MM-DD').format('MMMM Do, YYYY');
       contents.title = 'Please review your Supplemental Statement of the Case';
       contents.description = (
-        <p>The Veterans Benefits Administration sent you a Supplemental Statement of the Case on {formattedSocDate}
-        because, after completing the remand instructions from the Board, they couldn’t fully grant
+        <p>The Veterans Benefits Administration sent you a Supplemental Statement of the Case on {formattedSocDate} because,
+          after completing the remand instructions from the Board, they couldn’t fully grant
```
Comment (cehsu) [inline src/js/claims-status/utils/appeals-v2-helpers.jsx]: it looks like there's an extra space here now.

**Review by cehsu — APPROVED** [review]
LGTM. Should any tests be updated?

File: src/js/claims-status/utils/appeals-v2-helpers.jsx (hunk 1 above)
Comment (cvalarida) [reply src/js/claims-status/utils/appeals-v2-helpers.jsx]: Oh, I should mention why this was done: I opened up an appeal and saw the date backed right into the next word without a space, so I just bumped the next word up a line to make sure it was parsed with a space.

**Comment by cvalarida** [comment]
Turns out there weren't any tests for `isolateAppeals`, so I added some. Good thing, too; I had the parameter order wrong! :grimacing: Thanks for catching that, @cehsu :raised_hands:

File: test/claims-status/utils/helpers.unit.spec.js [hunk 2]
```diff
@@ -646,4 +647,34 @@ describe('Disability benefits helpers: ', () => {
       expect(descText).to.equal('Once your representative has completed their review, your case will be returned to the Board. A Veterans Law Judge, working with their team of attorneys, will review all of the available evidence and write a decision. For each issue you’re appealing, they can decide to:Grant: The judge disagrees with the original decision and decides in your favor.Deny: The judge agrees with the original decision.Remand: The judge sends the issue back to the Veterans Benefits Administration to gather more evidence or to fix a mistake before deciding whether to grant or deny.Note: About 60% of all cases have at least 1 issue remanded.');
     });
   });
+
+  describe('isolateAppeal', () => {
```
Comment (markgreenburg) [inline test/claims-status/utils/helpers.unit.spec.js]: hooray for tests!

**Review by markgreenburg — APPROVED** [review]
Looks great! Good idea to add some tests.

**Pull request merged**